### ISSUE TITLE
can not bind the address like "localhost"

### ISSUE THREAD

**Issue by cyqw** [tporadowski/redis#48]
I want start redis server on "locahost" (may be have ipv6 only host in the network, so do not use "127.0.0.1"), but failed.
I found BindParams::IsIPAddress can not support "localhost", is only support IP address.

**Comment by tporadowski** [comment]
Yes, that function relies on [WSAStringToAddressA](https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-wsastringtoaddressa) which handles only string representations of IPv4 and IPv6 addresses. You can therefore also use "::1" for IPv6 equivalent of localhost.

In original Redis I can see that `getaddrinfo` is used, which supports name lookups, so this might actually be a future enhancement to Redis for Windows.
